roles/openshift_master/tasks/main.yml

```diff
@@ -104,3 +104,14 @@
   with_items:
   - root
   - "{{ ansible_ssh_user }}"
+
+- name: Update the permissions on the OpenShift client config(s)
+  file:
+    path: "~{{ item }}/.config/openshift/.config"
```

Comment:
It is indeed ~/.config/openshift/.config.  It definitely doesn't pass the visual parser...